src/commands/support/trythis.ts

```diff
@@ -1,18 +1,22 @@
-import { CommandDefinition } from '../../lib/command';
 import { CommandCategory } from '../../constants';
-import { makeEmbed } from '../../lib/embed';
+import { MessageCommandDefinition } from '../../lib/command';
+import { makeEmbed, makeLines } from '../../lib/embed';
 
-export const trythis: CommandDefinition = {
+const trythisDescription = makeLines([
+    'Please try and remove all other mods/liveries from the community folder and test our addon again. This will help rule out mod conflicts.',
+    '',
+    'See [this article](https://docs.flybywiresim.com/fbw-a32nx/support/reported-issues/#test-with-only-the-a32nx-add-on-in-community) on how to this quickly.',
```

Comment:
```suggestion
    'See [this step in our troubleshooting guide](https://docs.flybywiresim.com/fbw-a32nx/support/reported-issues/#test-with-only-the-a32nx-add-on-in-community) for more details on how to do this easily.',
```
There was a verb missing in `on how to (do) this quickly.`. I thought the above language is a little clearer.

Reply:
Lol I rewrote that several times because I didn't like the wording...must have gotten lost in the process somewhere.